x-pack/plugins/monitoring/public/application/pages/use_monitoring_time.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useCallback, useState } from 'react';
+import createContainer from 'constate';
+
+interface TimeOptions {
+  from: string;
+  to: string;
+  interval: string;
+}
+
+export const DEFAULT_TIMERANGE: TimeOptions = {
+  from: 'now-1h',
+  to: 'now',
+  interval: '>=10s',
+};
+
+export const useMonitoringTime = () => {
+  const defaultTimeRange = {
+    from: 'now-1h',
+    to: 'now',
+    interval: DEFAULT_TIMERANGE.interval,
+  };
+  const [refreshInterval, setRefreshInterval] = useState(5000);
```

Comment:
This value [should be 10000](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/plugin.ts#L174).